docs/user-defined-redirects.rst

```diff
@@ -81,20 +95,41 @@ The example configuration would be::
 
 Your users query would now redirect in the following manner::
 
-	docs.example.com/dev/install.html ->
-	docs.example.com/en/latest/installing-your-site.html
+        docs.example.com/dev/install.html ->
+        docs.example.com/en/latest/installing-your-site.html
 
 Note that you should insert the desired language for "en" and version for "latest" to
 achieve the desired redirect.
 
+*Exact Redirects* could be also useful to redirect a whole sub-path to a different one by using a special ``$rest`` keyword in the "From URL".
+Let's say that you want to redirect your readers of your version ``2.0`` of your documentation under ``/en/2.0/`` because it's deprecated,
```

Comment:
This useful when renaming dirs too

Reply:
When renaming dirs the `$rest` argument shouldn't be used. Instead, you should just use `/old-dir/` to `/new-dir/`. This way, it will do this under all of the languages and versions.